pandunia-loge.csv

```diff
@@ -207,6 +207,10 @@ aventurer||||adventurer|aventurier|aventurero|aventureiro|авантюрист||
 avgani||||Afghan|afghan|afgano|afegão|афганский||||||अफगानी|||||afgana|afgaani|afgański
 Avganistan|desh|AF||Afghanistan|Afghanistan|Afganistán|Afeganistão|Афганистан||阿富汗|アフガニスタン|아프가니스탄||अफ़्ग़ानिस्तान||Afghanistan||Afganistan|Afganujo, Afganio|Afganistan|Afganistan
 awakate|biw||nah:ahuakatl, spa:aguacate, por:abacate, tgl:abokado, jpn:アボカド (abokado), kor:아보카도 (abokado), fra:avocat, pol:awokado, kon:zavoká|avocado||aguacate|||||||||||||avokado|avokado|awokado
+awal||||beginning (start)||comienzo (inicio)|||||始め|||शुरू|শুরু|awal (mula)|awali (mwanzo)|başlangıç|komenco|alku|początek
+awala||||begin (start, commence, initiate)||comenzar (iniciar, empezar)||||开始||||||||başlamak|komenci|aloittaa|zacząć, zaczynać
+awali|||ara:أَوَّل‎ (ʾawwal), may:awal, swa:awali, tur:evvel, fas:اول‎ (avval), hin:अव्वल (avval), ben:আউয়াল (auwal)|initial (first)||primer (inicial, inaugural)||||||||||||||ensimmäinen (alku-)|pierwszy, początkowy
+awalu||||begin (get started)||comenzar (iniciarse, empezar)|||||||||||||komenciĝi|alkaa|zaczyna (zacząć, rozpoczynać, rozpocząć)
```

Comment:
yi logasle no tena sahi men.

auwal : early; earlier; first [Mal ewwel, Aram awla (beginning)] Aze evvel, Hin avval, Ind awal, Per awwal, Swa awali, Tur evvel borrowed from Ara 

me no suku "kaish-" sabu le si jamloge da dul chini loge. me volu uza mono unfeni chini loge. mome ablu uza "mul-", sabu li men sa malayi bash e multi barti bash, a -- sa wi hal -- mome nidu novi asle pa "root".

Reply:
sa malayi e suahili le mena "beginning". a si, sa ali bash, le mena lilo alsifo.

me suku "mul" da mena "beginning" e "gen" da mena "root".

Reply:
me no shina "mul" mena "start" sa barti bash; le mena "root" o "origin". vero, rupu ko barti loge da mena "start" si "shuru". sa sabu we, me suja "mul" restu ko "root", e "shur" chanu ko "start".

Reply:
me okeya.